lib/travis/build/script/nix.rb

```diff
@@ -43,19 +43,13 @@ def configure
         def setup
           super
 
-          version = config[:nix]
+          version = config[:nix] == "latest" ? config[:nix] : "nix-#{config[:nix]}"
 
           sh.fold 'nix.install' do
-            sh.cmd "wget --retry-connrefused --waitretry=1 -O /tmp/nix-install https://nixos.org/releases/nix/nix-#{version}/install"
+            sh.cmd "wget --retry-connrefused --waitretry=1 -O /tmp/nix-install https://nixos.org/releases/nix/#{version}/install"
             sh.cmd "yes | sh /tmp/nix-install"
```

Comment:
```suggestion
            sh.cmd "yes | sh /tmp/nix-install --daemon"
```

I think this is the gap here.

Reply:
yes that should be sufficient